.travis.yml

```diff
@@ -8,8 +8,21 @@ php:
   - 5.6
   - hhvm
 
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: GUZZLE_VERSION=3.*
+    - php: 5.6
+      env: GUZZLE_VERSION=4.*
+    - php: 5.6
+      env: GUZZLE_VERSION=5.*
+    - php: 5.6
+      env: GUZZLE_VERSION=6.*
```

Comment:
well, but this means you run 3 more jobs per build, and KnpLabs has many open-source repos sharing this 5 concurrent jobs quota.